content/Properties/Properties.q.md

```diff
@@ -0,0 +1,1077 @@
+
+ - It accesses the `$address` property of the `Person` class.
+
+ - It declares and initializes the `$address` property of the `Person` class.
+
+
+/// type=SS, answer=[4]
+
+On line 12, what does the statement `$pObject = new Person();` do?
+
+ - It sets the value of `$pObject` to `Person`.
+
+ - It replaces the value of `$pObject` with `Person`.
+
+ - It accesses the value of `Person` through `$pObject`.
+
+ - It creates the `$pObject` object of the `Person` class.
+
+ - It assigns the value of the `Person` class to `$pObject`.
+
+
+/// type=SS, answer=[5]
+
+In the statement `echo $pObject->address;` on line 13, what does `$pObject->address` do?
+
+ - It sets the value of the `address` property of `$pObject`.
+
+ - It returns the value of the `address` property of `$pObject`.
+
+ - It removes the value of the `address` property of `$pObject`.
+
+ - It displays the value of the `address` property of `$pObject`.
+
+ - It accesses the value of the `address` property of `$pObject`.
+
+:::
+
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+    class Person 
+    {
+        public $name = "Diana";
+        public $address;
+        
+        public function eat()
+        {
+            echo "This is an eat() method.";
+        }
+    }
+    $pObject = new Person();
+    $pObject->address = "Canada";
+    echo $pObject->address;
+?>
+```
+/// type=SS, answer=[2]
+
+Execute the program. What is its output?
+
+ - It prints `Diana`.
+
+ - It prints `Canada`.
+
+ - It prints `address`.
+
+ - No output is displayed.
+
+ - It prints `This is an eat() method.`.
+
+
+/// type=SS, answer=[5]
+
+Which of the following is an object?
+
+ - `$name`
+
+ - `Diana`
+
+ - `Person`
+
+ - `$address`
+
+ - `$pObject`
+
+
+/// type=SS, answer=[3]
+
+Which of the following is a class?
+
+ - `$name`
+
+ - `Diana`
+
+ - `Person`
+
+ - `$address`
+
+ - `$pObject`
+
+
+/// type=MS, answer=[1,3,4,5]
+
+Which of the following are keywords?
+
+ - `new`
+
+ - `Diana`
+
+ - `class`
+
+ - `public`
+
+ - `function`
+
+
+/// type=SS, answer=[5]
+
+Which of the following is an object instantiation?
+
+ - `public $address;`
+
+ - `class Person { }`
+
+ - `public $name = "Diana";`
+
+ - `echo $pObject->address;`
+
+ - `$pObject = new Person();`
+
+
+/// type=SS, answer=[1]
+
+In the statement `$pObject->address = "Canada";` on line 13, what is `Canada`?
+
+ - It is a string.
+
+ - It is a method.
+
+ - It is a keyword.
+
+ - It is a property.
+
+ - It is a class name.
+
+
+/// type=SS, answer=[5]
+
+In the statement `$pObject->address = "Canada";` on line 13, what does the object operator `->` do?
+
+ - It sets the `address` property of `$pObject`.
+
+ - It creates the `address` property of `$pObject`.
+
+ - It defines the `address` property of `$pObject`.
+
+ - It declares the `address` property of `$pObject`.
+
+ - It accesses the `address` property of `$pObject`.
+
+
+/// type=SS, answer=[1]
+
+On line 13, what does the statement `$pObject->address = "Canada";` do?
+
```

Comment:
Kindly check the lengths of the options in this question. Thanks.
![PropertiesLength01](https://user-images.githubusercontent.com/38550259/61015515-890daf80-a3be-11e9-9aa4-b00243dbbfac.jpg)

Reply:
Checked this one already and I did not found anything about the alignment of the options. As shown in the image, the options are arranged according to their lengths. Please refer to the red line. Thanks. 
![image](https://user-images.githubusercontent.com/25945790/61016424-29190800-a3c2-11e9-95c9-327d0ba5d027.png)